pkgsettings/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .pkgsettings import DuplicateConfigureWarning, PrefixedSettings, Settings, SimpleSettings
+from .pkgsettings import DuplicateConfigureWarning, PrefixedSettings, Settings
 
-__all__ = ["DuplicateConfigureWarning", "PrefixedSettings", "Settings", "SimpleSettings"]
```

Comment:
This is may be considered a breaking change, however, no public API exposes parameters or return values of `SimpleSettings` directly. So, I believe it's not breaking

Reply:
AFAIK `__all__` is for doing `from X import *`, if someone is doing that in their repo... better to break it 🤣